terra-jupyter-base/custom/edit-mode.js

```diff
@@ -106,21 +106,21 @@ define(() => {
             return;
         }
 
-        checkMeta();
+        checkMeta(false);
         initSyncMaintainer();
     }
 
     function initSyncMaintainer() {
         syncMaintainer = setInterval(() => {
-            checkMeta();
+            checkMeta(true);
         }, lastLockedTimer);
 
         window.onbeforeunload(function() {
             clearInterval(syncMaintainer);
         });
     }
 
-    function checkMeta() {
+    function checkMeta(shouldStaggerCalls = true) {
```

Comment:
needed because we want the first lock call to occur immediately after check meta, and subsequent calls to be staggered